src/components/organisms/AssetContent/index.tsx

```diff
@@ -91,6 +104,16 @@ export default function AssetContent(props: AssetContentProps): ReactElement {
                   <Button style="text" size="small" onClick={handleEditButton}>
                     Edit Metadata
                   </Button>
+                  <span className={styles.separator}>|</span>
```

Comment:
separator should be part of the condition in next line, no need to show it when only 1 button exists